PyRDF/backend/AWS.py

```diff
@@ -82,8 +103,26 @@ def encode_object(object_to_encode) -> str:
             Value=str(pickled_reducer),
             Overwrite=True
         )
+        """
+
+        def invoke_root_lambda(root_range, script):
+
+            trials = 3
+
+            client = boto3.client('lambda', region_name=self.region)
+            def process_response(response):
```

Comment:
this whole function could be simplified to "try json.loads(response[payload].read) except exception e debug log and set empty dict"